manuscript/mtls/README.md

```diff
@@ -16,19 +16,19 @@ Network Policies and mTLS are often added to a Kubernetes cluster using an appli
 
 ## Choice 1: Istio
 
-Like [Ingress](manuscript/ingress/README.md), the L7 service mesh technologies (Linkerd, Istio, Kuma) each act as a *control plane* that does the complicated work of configuring and managing an underlying networking technology (often referred to as the *data plane*). This data plane can also be called a *networking layer*, *service proxy*, or just *proxy*. The job of the data plane is to add security, capture usage data, and, most importantly, actually route the requests.
+Like [Ingress](manuscript/ingress/README.md) technologies, the L7 service mesh technologies (Linkerd, Istio, Kuma) each act as a *control plane* that does the complicated work of configuring and managing an underlying networking technology (often referred to as the *data plane*). This data plane can also be called a *networking layer*, *service proxy*, or just *proxy*. The job of the data plane is to add security, capture usage data, and, most importantly, actually route the requests.
```

Comment:
This and the next paragraph are generic and not necessarily applicable only to Istio (it even mentions others). Would it make sense to move it up to the intro?

Reply:
I hear what you're saying. The statements are about the app layer meshes so they're in between - they apply to 3/5 of the technologies in this section.

I'm not sure that I want to open that can of worms in the intro, especially when it doesn't apply to everything. Maybe we have a separate intro for the L7 meshes. Let's talk it out during preshow today.